docs/maintain/wasp/v1.0.0-rc.6/docs/how-tos/running-an-access-node.md

```diff
@@ -0,0 +1,252 @@
+
+### Download the Chain State
+
+You can configure how much of the state you wish to keep on your node using the following setting: `stateManager.pruningMinStatesToKeep`. The default value is 10000 blocks.
+
+### Run a Non-archive Node
+
+If you wish to run a non-archive node (only keep the last N blocks), you must add `snapshots.networkPaths` to your config:
+
+<Tabs groupId="network" queryString>
+<TabItem value="shimmer" label="ShimmerEVM">
+
+```diff docker-compose.yml
+       - "--wal.path=/app/waspdb/wal"
+       - "--prometheus.enabled=true"
+       - "--prometheus.bindAddress=wasp:9312"
++      - "--stateManager.pruningMinStatesToKeep=10000"
++      - "--snapshots.networkPaths=https://files.shimmer.network/?prefix=wasp_snapshots/smr1prxvwqvwf7nru5q5xvh5thwg54zsm2y4wfnk6yk56hj3exxkg92mx20wl3s/"
+     profiles:
+       - wasp
+```
+
+</TabItem>
+<TabItem value="shimmer_testnet" label="ShimmerEVM Testnet">
+
+```diff docker-compose.yml
+       - "--wal.path=/app/waspdb/wal"
+       - "--prometheus.enabled=true"
+       - "--prometheus.bindAddress=wasp:9312"
++      - "--stateManager.pruningMinStatesToKeep=10000"
++      - "--snapshots.networkPaths=https://files.testnet.shimmer.network/?prefix=wasp_snapshots/rms1ppp00k5mmd2m8my8ukkp58nd3rskw6rx8l09aj35984k74uuc5u2cywn3ex/"
+     profiles:
+       - wasp
+```
+
+</TabItem>
+</Tabs>
+
+### Run An Archive Node
+
+If you wish to have a full archive node you need to set `stateManager.pruningMinStatesToKeep` to `0` and can remove the `snapshots.networkPaths` option. 
+
+You can then download the historical state using the following command (this will take a while):
+
+
+<Tabs groupId="network" queryString>
+<TabItem value="shimmer" label="ShimmerEVM">
+
+```sh
+wget https://files.shimmer.network/dbs/wasp/latest-wasp_chains_wal.tgz -O - | tar xzv -C data/wasp
+```
+
+</TabItem>
+<TabItem value="shimmer_testnet" label="ShimmerEVM Testnet">
+
+```sh
+wget https://files.testnet.shimmer.network/dbs/wasp/latest-wasp_chains_wal.tgz -O - | tar xzv -C data/wasp
+```
+
+</TabItem>
+</Tabs>
+
+:::note Disk Space
+
+Operating as a full archive node requires a lot of disk space. We recommend at least 500Gb of free space to operate without issues
+
+:::
+
+## Run Everything
+
+```sh
+docker compose up -d
+```
+
+It will take a few minutes until the hornet node is synced.
+
+You can check the sync status by following the logs with `docker logs -f hornet`, or in the web dashboard.
+
+## Download Wasp-Cli
+
+You can download a Wasp-Cli that matches your Wasp version from the [Wasp releases](https://github.com/iotaledger/wasp/releases).
+You can use a command like this to download the Wasp-Cli:
+
+```sh
+curl -sL https://github.com/iotaledger/wasp/releases/download/v1.0.3/wasp-cli_1.0.3_Linux_x86_64.tar.gz | tar xzv
+```
+
+## Use Wasp-Cli to Obtain Peering Info
```

Comment:
This header feels like it's on the wrong level, or at least the headers below are on the wrong level. The paragraph below only describes how to `cd` into the cli directory, actually getting the peering info is described way down. Should the following headers be one level down?

Reply:
Yeah makes sense. I think we should actually change the header to something else. The whole section should be called setting up with wasp cli or something and everything else should be sub steps I guess. Let me commit something

Reply:
603bf9390111a3dbc444e20f7939cf72462233a5 Wdyt?